plasma_framework/test/src/exits/payment/PaymentStartInFlightExit.test.js

```diff
@@ -324,28 +338,212 @@ contract('PaymentInFlightExitRouter', ([_, alice, richFather, carol]) => {
                 this.startIFEBondSize = await this.exitGame.startIFEBondSize();
             });
 
-            it('should fail when spending condition not registered', async () => {
-                const {
-                    args,
-                    inputTxsBlockRoot1,
-                    inputTxsBlockRoot2,
-                } = buildValidIfeStartArgs(
-                    AMOUNT,
-                    [alice, bob, carol],
-                    [OUTPUT_TYPE_ONE, OUTPUT_TYPE_ONE, OUTPUT_TYPE_ONE],
-                    BLOCK_NUMBER,
-                    DEPOSIT_BLOCK_NUMBER,
-                );
-                await this.framework.setBlock(BLOCK_NUMBER, inputTxsBlockRoot1, 0);
-                await this.framework.setBlock(DEPOSIT_BLOCK_NUMBER, inputTxsBlockRoot2, 0);
-
-                await expectRevert(
-                    this.exitGame.startInFlightExit(args, { from: alice, value: this.startIFEBondSize.toString() }),
-                    'Spending condition contract not found',
-                );
+            describe('and output guard handlers for all output types are registered', () => {
```

Comment:
minor: I would start with `given` instead of `and` here but that might be just my preference